unlock-app/src/components/interface/checkout/alpha/Checkout/Select.tsx

```diff
@@ -254,37 +255,56 @@ export function Select({ checkoutService, injectedProvider }: Props) {
                                   />
                                 </div>
                               </div>
-                              <div className="flex items-center justify-between w-full">
-                                <div className="flex items-center gap-2">
-                                  <LabeledItem
-                                    label="Duration"
-                                    icon={DurationIcon}
-                                    value={formattedData.formattedDuration}
-                                  />
-                                  <LabeledItem
-                                    label="Quantity"
-                                    icon={QuantityIcon}
-                                    value={
-                                      formattedData.isSoldOut
-                                        ? 'Sold out'
-                                        : formattedData.formattedKeysAvailable
-                                    }
-                                  />
+
+                              <div className="w-full space-y-2">
+                                <div className="flex items-center w-full grid-cols-2 gap-2">
+                                  {item.recurringPayments && (
+                                    <Badge variant="primary" size="tiny">
+                                      Renew{' '}
+                                      {typeof item.recurringPayments ===
+                                      'number'
+                                        ? `${item.recurringPayments} times`
+                                        : item.recurringPayments}
+                                    </Badge>
+                                  )}
+                                  {isMember && (
+                                    <Badge size="tiny" variant="primary">
+                                      Owned
```

Comment:
I think it is a great idea to show to the user when they already have a membership to a lock, but I am not sure this UI is the right one :/